tutorials/FSS.ipynb

```diff
@@ -0,0 +1,2329 @@
+      ],
+      "text/plain": [
+       "<xarray.DataArray (window_size: 5, threshold: 5, scenario: 3)> Size: 600B\n",
+       "array([[[0.9975056 , 0.8781663 , 0.99752491],\n",
+       "        [0.99610463, 0.81739954, 0.99049211],\n",
+       "        [0.99433897, 0.74370923, 0.96275137],\n",
+       "        [0.99147185, 0.6619713 , 0.90505197],\n",
+       "        [0.9868625 , 0.57376777, 0.80921571]],\n",
+       "\n",
+       "       [[0.99937841, 0.87859082, 0.99939846],\n",
+       "        [0.99896359, 0.8178622 , 0.99287998],\n",
+       "        [0.99845944, 0.7439784 , 0.96542867],\n",
+       "        [0.99769299, 0.66258674, 0.90807087],\n",
+       "        [0.99678288, 0.57443367, 0.8125889 ]],\n",
+       "\n",
+       "       [[0.99973902, 0.87864503, 0.99974371],\n",
+       "        [0.99955579, 0.81798018, 0.99333381],\n",
+       "        [0.99935263, 0.74364957, 0.96569631],\n",
+       "        [0.99896652, 0.66246346, 0.90825963],\n",
+       "        [0.99866962, 0.57467106, 0.81326944]],\n",
+       "\n",
+       "       [[0.9998519 , 0.87883252, 0.99986309],\n",
+       "        [0.99975721, 0.81818033, 0.99351962],\n",
+       "        [0.99966944, 0.74355962, 0.96581436],\n",
+       "        [0.99942939, 0.66246222, 0.90830549],\n",
+       "        [0.99928144, 0.57516051, 0.81375132]],\n",
+       "\n",
+       "       [[0.99990218, 0.87900485, 0.99991912],\n",
+       "        [0.99984607, 0.81846299, 0.99364039],\n",
+       "        [0.99980318, 0.74363044, 0.96587942],\n",
+       "        [0.9996493 , 0.66234939, 0.90827159],\n",
+       "        [0.99954734, 0.5753859 , 0.81391288]]])\n",
+       "Coordinates:\n",
+       "  * window_size  (window_size) <U9 180B '[20 20]' '[40 40]' ... '[100 100]'\n",
+       "  * threshold    (threshold) <U4 80B '0.0' '0.25' '0.5' '0.75' '1.0'\n",
+       "  * scenario     (scenario) int64 24B 0 1 2\n",
+       "Attributes:\n",
+       "    description:  Fractions skill score"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import xarray as xr\n",
+    "import itertools\n",
+    "import numpy as np\n",
+    "\n",
+    "# as before\n",
+    "window_sizes = np.linspace((20,20), (100,100), 5, dtype=(int,int))\n",
+    "thresholds = np.linspace(0.0, 1.0, 5, dtype=float)\n",
+    "input_scenarios = [[obs, fcst_1], [obs, fcst_2], [obs, fcst_3]]\n",
+    "compute_method = FssComputeMethod.NUMPY\n",
+    "\n",
+    "# create output array\n",
+    "W = len(window_sizes)\n",
+    "T = len(thresholds)\n",
+    "S = len(input_scenarios)\n",
+    "fss_out = np.zeros((W, T, S))\n",
+    "\n",
+    "# lets iterate over the indices, setting the `multi_index` and `writeonly` flags\n",
+    "with np.nditer(fss_out, flags=['multi_index'], op_flags=['writeonly']) as it:\n",
```

Comment:
Yes, but not yet. I think it should be addressed in #353 or #269 . Because:
- I'd like a more efficient version of doing it and that requires some thought.
- This PR already is getting quite heavy.

However, if there's a strong requirement (or is probably the most common use-case) for it to be part of the PR I'll consider it.

Reply:
I'm gearing towards differing this to a later issue if that's okay...